src/infra/Tags/Tags.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Tag from './Tag/Tag';
+
+function Tags(props) {
```

Comment:
```suggestion
function Tags({ tags }) {
```

i think when u have few props is easier for the eyes to just destruct the props